src/components/context-selection/OrgUnitSelector.js

```diff
@@ -49,41 +51,44 @@ export const OrgUnitSelector = ({
             setOpen={setOpen}
             noValueMessage={'Choose an organisation unit'}
         >
-            <div className={styles.container}>
-                <div className={styles.orgUnitTreeContainer}>
-                    <span className={styles.orgUnitSpan}>
-                        Choose an organisation unit
-                    </span>
-                    <OrganisationUnitTree
-                        singleSelect
-                        onChange={(orgUnit, e) => {
-                            e.stopPropagation()
-                            // level is not included on selected; would be better to amend
-                            const computedLevel = (
-                                orgUnit.path.match(/\//g) || []
-                            ).length
-                            // clear out selected level if selected org unit is too low
-                            if (
-                                selectedOrgUnitLevel &&
-                                Number(selectedOrgUnitLevel) <= computedLevel
-                            ) {
-                                setSelectedOrgUnitLevel(null)
-                            }
-                            setSelectedOrgUnit({
-                                ...orgUnit,
-                                level: computedLevel,
-                            })
-                        }}
-                        isUserDataViewFallback={true}
-                        roots={rootOrgUnits}
-                        selected={
-                            selectedOrgUnit?.path ? [selectedOrgUnit?.path] : []
-                        }
-                    />
-                </div>
-                <div className={styles.orgUnitTreeContainer}>
+            <div className={styles.menuContainer}>
```

Comment:
For this component, I aligned the menu styles with what's in the new Data Entry app (and did the same for the period selector)